Dockerfile

```diff
@@ -9,7 +9,7 @@ COPY deployments/crds ./deployments/crds
 ENV CGO_ENABLED=0 GOOS=linux
 RUN go build -o hephaestus-controller ./cmd/controller
 
-FROM gcr.io/distroless/static:nonroot
+FROM gcr.io/distroless/static:debug-nonroot
```

Comment:
Why `debug` and not just `non-root`?

Reply:
that will be reverted... when debugging... I needed access to a shell to see what is going on.

Reply:
reverted